tests/integration/integration_test_case.py

```diff
@@ -452,6 +452,20 @@ def assertInRedirect(self, content):
         else:
             self.fail("no redirect found")
 
+    def launchAndFailSubmission(self, schema):
+        self.launchSurvey(schema)
+        self.post()
+        self.post()
+        self.post()
+
+    def getUrlAndCookie(self, url):
+        self.get(url=url)
+        return self.getCookie()
+
+    def deleteCookieAndGetUrl(self, url):
+        self.deleteCookie()
+        self.get(url=url)
```

Comment:
This file should only contain common methods. These seem specific, if so they should live in the test file.

Reply:
Moved them across but kept naming convention should we change that as well?